src/openapi-generator/serializable/adapters/active-model.cr

```diff
@@ -0,0 +1,38 @@
+require "open_api"
+require "../serializable"
+require "../../extensions"
+require "active-model"
+
+module OpenAPI::Generator::Serializable::Adapters::ActiveModel
+  {% unless (@type == OpenAPI::Generator::Serializable::Adapters::ActiveModel || @type.ancestors.includes?(::ActiveModel::Model)) %}
+    raise AdapterError.new("ActiveModel::Model was not inherited for type {{@type}}")
```

Comment:
Could make this a static error so it happens at compile time rather than run-time

Reply:
How can I do that?

``crystal
{% raise Error.new %}
```
?